README.md

```diff
@@ -1364,6 +1364,43 @@ function MyComponent() {
 }
 ```
 
+`graphql-hooks` also supports `TypedDocumentNode`. This allows you to use GraphQL code gen to create `DocumentNode`s for your GQL queries and receive full type support.
```

Comment:
let's make sure we also document the simple version without the types, by using `graphql-tag`

Reply:
I've added a section on Document node usage